packages/schemas/src/db-entries/custom-types.ts

```diff
@@ -11,6 +11,13 @@ export enum SignAlgorithmType {
 export enum AccessTokenFormatType {
   jwt = 'jwt',
 }
+export enum TenantRegionType {
```

Comment:
just for further development: we should use [ISO-3166](https://en.wikipedia.org/wiki/ISO_3166) compatible way to define country / region. maybe we can find a lib later.